recipe/run_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -xe
+
+# ensure the pkgconfig files are installed correctly
+pkg-config --print-provides "$PKG_NAME"
```

Comment:
The contents of this need to be moved into the meta.yaml.  I forgot that I had created a `run_test.sh` on the old branch.

Reply:
resolved in 
e7c1aee